how-to/workspace-platform-starter/client/src/framework/workspace/microflow.ts

```diff
@@ -0,0 +1,120 @@
+import { Integrations } from "@openfin/workspace";
+import type { WorkflowIntegration } from "@openfin/workspace-platform";
+import { createLogger } from "../logger-provider";
+import type { MicroflowProviderOptions } from "../shapes/microflow-shapes";
+import { isEmpty } from "../utils";
+
+const logger = createLogger("Microflows");
+
+let microflowOptions: MicroflowProviderOptions | undefined;
+let microflows: { [key: string]: WorkflowIntegration };
+const requiresSearchInitialization: string[] = [];
+
+/**
+ * This function uses the passed configuration to create an instance of each required microflow and
+ * creates a list of microflows that still need their search integration initialized.
+ */
+async function initializeMicroflows(): Promise<void> {
+	const microflowDefinitions = microflowOptions?.microflows || [];
```

Comment:
Use optional chain, lint should have picked it up

Reply:
updated to use ??